custom-vu/src/main/kotlin/jces1209/vu/page/DcIssuePage.kt

```diff
@@ -57,4 +60,53 @@ class DcIssuePage(
     override fun linkIssue(): DcIssueLinking {
         return DcIssueLinking(driver)
     }
+
+    override fun changeAssignee(): DcIssuePage {
+        var assignee = "Tdztbchh Eskfsyas"
+
+        driver
+            .findElement(By.cssSelector("[id = 'assignee-val']"))
+            .click();
+
+        val userPiccker = driver
+            .wait(
+                ExpectedConditions.presenceOfElementLocated(By.xpath("//*[@id=\"assignee-form\"]"))
+            )
+
+        Actions(driver)
+            .sendKeys(Keys.DELETE, Keys.ARROW_DOWN)
+            .perform()
+
+        driver
+            .wait(
+                ExpectedConditions.presenceOfElementLocated(By.xpath("//*[@id=\"assignee-group-suggested\"]"))
+            )
+
+        var selectedUser = driver
+            .wait(
+                ExpectedConditions.elementToBeClickable(By.xpath("//*[@id=\"all-users\"]/li[3]/a"))
+            )
+
+        var user = selectedUser.text.substringBefore(" - ")
+
+        Actions(driver)
+            .sendKeys(user)
+            .perform()
+
+        driver
+            .wait(
+                ExpectedConditions.presenceOfElementLocated(By.xpath(String.format("//a/em[.=\"%s\"]", user)))
```

Comment:
What does that selector represent? I'd extract the selector to a `val`, so it can be named.
`em` is a stylistic component, adds italics by default. We should locate by semantics, not style.
`a` is structural - it represents a link. It's better than relying on style, but it's still not semantics. Often, links are replaced with buttons or other interactive controls.

- [ ] avoid locating by style - `em`
- [ ] if possible, avoid locating by GUI implementation - `a`